includes/modules/pages/ask_a_question/header_php.php

```diff
@@ -24,7 +24,9 @@
 //
 $info_page = zen_get_info_page($pid);
 $show_info_page_ask_a_question = 'SHOW_' . strtoupper($info_page) . '_ASK_A_QUESTION';
-if (!defined($show_info_page_ask_a_question) || constant($show_info_page_ask_a_question) === '0') {
+$bypass_redirect = false;
+$zco_notifier->notify('NOTIFY_ASK_A_QUESTION_ALLOW_BYPASS_REDIRECT', array('products_id' => $pid, ), $bypass_redirect);
```

Comment:
I'm not sure where the "unilateral decision" comment came from (please post a reference), but for this PR to be accepted please use the short-array notation for the notifier's first parameter.